elections.md

```diff
@@ -0,0 +1,114 @@
+
+Nominators are free to nominate as many people as they wish to. Members of
+Standing may endorse multiple nominees, but we expect endorsements to be in
+good faith.  If this turns out to be a problem, this will be reconsidered.
+
+### Eligibility for voting
+
+All Members of Standing are eligible to vote for the steering committee
+members. 
+
+### Election process
+
+Elections will be held using time-limited Condorcet ranking on CIVS. The top
+vote getters will be elected to the respective positions. 
+
+### Maximal representation
+
+To encourage diversity there will be a maximum of one-third representation on
+the Steering Committee from any one company at any time. If the results of an
+election result in greater than 1/3 representation, the lowest vote getters
+from any particular company will be removed until representation on the
+committee is less than one-third.
+
+If percentages shift because of job changes, acquisitions, or other events,
+sufficient members of the committee must resign until max one-third
+representation is achieved. If it is impossible to find sufficient members to
+resign, the entire company’s representation will be removed and new special
+elections held. In the event of a question of company membership (for example
+evaluating independence of corporate subsidiaries) a majority of all
+non-involved Steering Committee members will decide. 
+
+### Initial Election
+
+Because of the need to bootstrap a staggered election cycle, some of the
+initial committee members will only serve a single year term. These three
+people will be the lowest vote getters from the top six in the initial
+election.
+
+The bootstrap committee will operate the election and circulate a timeline for
+nominations, and the vote.
+
+### Vacancies
+
+In the event of a resignation or other loss of a steering committee member, the
+candidate with the next most votes from the previous election will be offered
+the seat.  This process will continue until the seat is filled.  In case this
+fails to fill the seat, a special election for that position will be held as
+soon as possible. The same group of people as described in “eligibility for
+voting” will vote in the special election. A committee member elected in a
+special election will serve out the remainder of the term for the person they
+are replacing, regardless of the length of that remainder. If a continuity
+member resigns or otherwise is lost from the Committee, that position will not
+be filled.
+
+### Limiting Corporate Campaigning
+
+To reduce size of company advantages, candidates may not use their companies
+internal or external brand to campaign.  Their employers cannot solicit votes
+on their behalf or sponsor candidates from partner organizations.  Simply put,
+elections highlight individuals outside of their corporate role and should be
+treated as “brand free” activities.
+
+## Refactoring or reforming the steering committee
```

Comment:
i think this should move into charter.md and out of elections.md.

Reply:
and maybe this just becomes "refactoring or reforming the charter".

Reply:
This belongs more in a voting or process doc then the charter imo.

Reply:
Because the charter gives the steering committee the power to set elections, and more steering committee is part of the process?